apollo-router/src/configuration/mod.rs

```diff
@@ -241,9 +238,8 @@ impl Configuration {
         cors: Option<Cors>,
         plugins: Map<String, Value>,
         apollo_plugins: Map<String, Value>,
-        dev: Option<bool>,
```

Comment:
Suggest leaving the dev option on the builder. Then you can avoid modifying the env variable in your test.

Reply:
It won't fix the issue with en variables in my tests. Because I provide a yaml file and then it's directly deserialized. I don't use the builder

Reply:
Fixed